test/tests.bats

```diff
@@ -1423,6 +1442,12 @@ load 'test_helper/bats-assert/load'
 
 @test "checking setup.sh: email add login validation" {
   # validates that the user created previously with setup.sh can login
+
+  if ! (docker exec mail doveadm auth test -x service=smtp setup_email_add@example.com 'test_password' >/dev/null); then
```

Comment:
could probably also be simplified with https://stackoverflow.com/a/12321815/4754613

Reply:
Yes, but again I'd like to keep it simple as I'm not that good with bash. Feel free to improve the code though.